tests/functional/catalogue/test_catalogue.py

```diff
@@ -1,13 +1,29 @@
+import json
```

Comment:
Think these tests should go in `functional/dashboard/test_catalogue/py` instead of `functional/catalogue`